kuma/wiki/forms.py

```diff
@@ -442,7 +443,20 @@ def akismet_error(self):
             document = self.instance.document
         except ObjectDoesNotExist:
             document = None
-        # wrapping this in a try/finally to make sure that even if
+
+        if exception and isinstance(exception, AkismetError):
+            # For Akismet errors, save the submission and exception details
+            dsa_params = parameters.copy()
+            dsa_params['_akismet_status_code'] = exception.status_code
```

Comment:
does this need to be arbitrarily prefixed with `_` here? I mean, what does that mean, is this "private" data?